### DISCUSSION TITLE
addGeneExpression with option to remove cells that do not exist in seRNA

### DISCUSSION THREAD

**Discussion by jmmuncie** [GreenleafLab/ArchR#1450] [Feature Requests]
Sorry - I created this without realizing the `strictMatch` parameter had been added to `addGeneExpressionMatrix`...please ignore/delete.

**Comment by rcorces** [comment]
I know you've already edited your original post but...
The `strictMatch` parameter just checks for a strict match and then warns you if it doesnt exist. It doesnt currently automatically subset the ArchRProject because I think thats a pretty heavy-handed thing to do in the function (ie. requires electing a new name for the subsetted project etc).

https://github.com/GreenleafLab/ArchR/blob/7358dd109053049ae19ef8c64eb129e544805579/R/MatrixGeneExpression.R#L20-L22

I fixed some typos and made the `strictMatch` parameter mentioned in the warning that you received originally via https://github.com/GreenleafLab/ArchR/commit/e2d911bbe3df32afd8160194150b60d5514bc559. Let me know if you think more needs to be done
